docs/prebuilt/v2/prebuilt/Screens-and-components.mdx

```diff
@@ -22,82 +22,138 @@ By default, components are enabled based on the role associated with the use cas
 </video>
 
 ### Preview Screen
+The Prebuilt preview screen offers a set of components designed to enhance the initail set-up before joining a room. Choose a role to kickstart your customization journey.
 
-#### Header
+##### Header
 Set up a custom header for each roles using Title and Subtitle fields.
-#### Join
+##### Join
 Choose how peers join the room
-	- **Join Now** - Peers from this role will see "Join Now" and would be able join after entering their name.
-	- **Join and Start Streaming** - Peers from this role will see "Go Live" and would be able to join and start streaming when they join the room. Ensure that this role has the permission to "Start/Stop HLS streaming" as `enabled` under Role permissions on its template configuration.
+	- **Join Now** - Peers from this role will see 'Join Now' and would be able join after entering their name.
+	- **Join and Start Streaming** - Peers from this role will see 'Go Live' and would be able to join and start streaming when they join the room. Ensure that this role has the permission to 'Start/Stop HLS streaming' as `enabled` under Role permissions on its template configuration.
+
+##### Skip Preview
+	- Turn the toggle to enable or disable skipping the preview screen.
+	  - **Enabled**: When this option is enabled, the preview screen is skipped, and the user is taken directly to the room screen.
+	  <Note> 
+	  When skip preview is enabled, ensure that the username is passed in the prebuilt links or in the options in the prebuilt component. If not provided, the system will generate a random UUID as the peer's name.
+	  </Note>
+
+##### Noise Cancellation State
+	- Turn the toggle to define the initial state for Noise Cancellation.
+	  - **Enabled**: When this option is enabled, Noise Cancellation will be turned on by default in preview screen for all the peers of this role.
+	  <Note> 
+	  Noise Cancellation should be enabled from 'Advanced Settings' under the Template to enable this setting. Learn more on enabling this feature from [here](/get-started/v2/get-started/features/noise-cancellation#enabling-the-noise-cancellation)
```

Comment:
Sentence update:

Learn more about enabling this feature [here](https://github.com/get-started/v2/get-started/features/noise-cancellation#enabling-the-noise-cancellation).